schemas/config-editor.xsd

```diff
@@ -247,15 +251,9 @@ If expanded, then one field per language is displayed with no need to click on t
         <![CDATA[
 Defining fields to display as tables
 ------------------------------------
-
```

Comment:
We can try removing blank lines as now the documentation is using markdown, I think it should be ok. But remember that this file was used to build the documentation (was sphinx syntax) - we still need to rework the conversion to the new manual in markdown